flask_lesson_intro/main.py

```diff
@@ -0,0 +1,31 @@
+from flask import Flask, render_template
+from utils import get_data
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def get_home_page():
+    return render_template("home.html")
+
+
+@app.route('/<item>')
+def get_item_page(item):
+    for i in get_data():
+        if i['title'] == item:
+            return render_template("item.html",
+                                   title=i['title'],
+                                   text=i['text'],
+                                   image=i['title'] + '.jpg',
+                                   count=len(i['text'].split()))
+        else:
```

Comment:
this else may be at one tabulation left?

Reply:
`for
...
else:`

Reply:
fixed - it was nonsense